build.ps1

```diff
@@ -121,5 +119,11 @@ if ($Test -and $PSCmdlet.ShouldProcess('Test assemblies')) {
         $xunitArgs += "-parallel","all"
     }
 
+    Write-Host "Testing x86..." -ForegroundColor Yellow
+    $xunitRunner = Join-Path $PackageRestoreRoot 'xunit.runner.console/2.4.1/tools/net472/xunit.console.x86.exe'
+    & $xunitRunner $xunitArgs
```

Comment:
When this has a failing test, the exit code from the ps1 script is still 0, which would cause the CI to succeed.